### PR TITLE
Fix clipboard issue (fixes #10453)

### LINKED ISSUE #10453: Docs: fix clipboard code snippets
We added a new copy button to code snippets in our docs. For the most part, this works great out of the box, but we need to check the snippets across the documentation to see if the clipboard output causes problems or errors.

copy-button

example: [Installation instructions](https://bedrock.readthedocs.io/en/latest/install.html)

Good
```
make build-prod run-prod
```

Bad
```
python3 -m venv venv                         # create a virtual env in the folder `venv`
source venv/bin/activate                     # activate the virtual env. On Windows, run: venv\Scripts\activate.bat
pip install --upgrade pip                    # securely upgrade pip
pip install -r requirements/dev.txt          # installs dependencies
```

This will require checking all the documentation pages for code snippets and using the copy button to confirm the output works as expected. ~If something breaks, move information out of the code snippet and into regular text or break the code snippet into multiple blocks.~

UPDATE (15-09-2021): after some [discussion](https://github.com/mozilla/bedrock/issues/10453#issuecomment-919427098), we have a few general rules we'd like to apply consistently:

1.    If the code is meant for the terminal, it should go one command per code block
2.   If the code is meant for a code editor, it can have multi-line code blocks (emphasis on the can, not should)
3.    Do not include comments in terminal command code, as it can cause errors

### PR DESCRIPTION
**Pull request by Ayushsunny**
## Description

Fixed all the clipboard code snippets 

## Issue / Bugzilla link

#10453

### CODE REVIEW

**Comment by Ayushsunny** [comment]
Hello @maureenlholland  , @reemhamz 

Could you please review this PR.
And sorry for the late.

File: docs/pipeline.rst [hunk 1]
```diff
@@ -92,10 +92,11 @@ to change according to the settings in the `.gitlab-ci.yml file in the www-confi
     are equivalent::
 
         $ bin/tag-release.sh --push -r mozilla
-        $ MOZ_GIT_REMOTE=mozilla bin/tag-release.sh --push
 
     And if you'd like to just tag and not push the tag anywhere, you may omit the ``--push``
-    parameter.
+    parameter ::
+
+        $ MOZ_GIT_REMOTE=mozilla bin/tag-release.sh --push
```
Comment (maureenlholland) [inline docs/pipeline.rst:99]: I think we want to keep this command under the `bin/tag-release.sh` command so it's still clear they are equivalent, but in separate code blocks so you can copy one or the other.

File: docs/install.rst [hunk 2]
```diff
@@ -87,14 +99,30 @@ These instructions assume you have Python 3.6+, pip, and NodeJS installed. If yo
 
 You need to create a virtual environment for Python libraries::
 
-    $ python3 -m venv venv                         # create a virtual env in the folder `venv`
-    $ source venv/bin/activate                     # activate the virtual env. On Windows, run: venv\Scripts\activate.bat
-    $ pip install --upgrade pip                    # securely upgrade pip
-    $ pip install -r requirements/dev.txt          # installs dependencies
+1.Create a virtual env in the folder `venv` ::
```
Comment (maureenlholland) [inline docs/install.rst:102]:
```suggestion
1. Create a virtual env in the folder `venv` ::
```

File: docs/install.rst [hunk 3]
```diff
@@ -87,14 +99,30 @@ These instructions assume you have Python 3.6+, pip, and NodeJS installed. If yo
...
+
+    $ python3 -m venv venv             
+
+2.Activate the virtual env. On Windows, run: venv\Scripts\activate.bat ::
```
Comment (maureenlholland) [inline docs/install.rst:106]:
```suggestion
2. Activate the virtual env. On Windows, run: venv\Scripts\activate.bat ::    
```

File: docs/install.rst [hunk 4]
```diff
@@ -87,14 +99,30 @@ These instructions assume you have Python 3.6+, pip, and NodeJS installed. If yo
...
+1.Create a virtual env in the folder `venv` ::
+
+    $ python3 -m venv venv             
+
+2.Activate the virtual env. On Windows, run: venv\Scripts\activate.bat ::    
 
-If you are on OSX and some of the compiled dependencies fails to compile, try explicitly setting the arch flags and try again::
+    $ source venv/bin/activate             
+
+3.Securely upgrade pip ::
```
Comment (maureenlholland) [inline docs/install.rst:110]:
```suggestion
3. Securely upgrade pip ::
```

File: docs/install.rst [hunk 5]
```diff
@@ -87,14 +99,30 @@ These instructions assume you have Python 3.6+, pip, and NodeJS installed. If yo
...
+
+    $ pip install --upgrade pip          
+
+4.Installs dependencies ::
```
Comment (maureenlholland) [inline docs/install.rst:114]:
```suggestion
4. Installs dependencies ::
```

**Review by maureenlholland — CHANGES_REQUESTED** [review]
@Ayushsunny Fantastic work! There are a lot of pages to check here and they are looking much better with the new code blocks. 

A couple of minor changes and one final code block to separate out, under Fluent: https://bedrock.readthedocs.io/en/latest/l10n.html#active-locales

It would be good to keep the following separate for copy/paste:
```
$ ./manage.py l10n_update
```
```
$ ./manage.py active_locales mozorg/mission
```

Then we can put the following block under the explanation text "You get an alphabetically sorted list..."
```
There are 91 active locales for mozorg/mission.ftl:
- af
- an
- ar
- ast
- az
- be
- bg
- bn
...
```

Hopefully this makes it clear that the long block of locales is an output example, and not something you want to copy/paste.

Let me know if you have any questions. Thanks again for picking this up!

**Comment by Ayushsunny** [comment]
Done @maureenlholland

**Review by maureenlholland — APPROVED** [review]

**Pull request merged**